src/tests/gtests.cpp

```diff
@@ -149,3 +146,25 @@ TEST_F(LeelaTest, KoSqNotSame) {
     // But ko (square) is not
     EXPECT_NE(hash, maingame.board.get_hash());
 }
+
+TEST_F(LeelaTest, TimeControl) {
+    // Initialize network with a small 32x1 network.
+    cfg_weightsfile = "../test_32x1.txt";
```

Comment:
@bood ok thanks for looking into the GPU issue. But another issue is this file cannot be found by Travis. It works in my local area because I do `mkdir build; cd build; make` and the file is in the dir above. On Travis the build directory is somewhere else so this doesn't work.
https://travis-ci.org/gcp/leela-zero/builds/331628882#L2888

Reply:
You can change the Dockerfile to do similar thing as well. Use following lines to replace `RUN CXX=g++ CC=gcc cmake CMakeLists.txt`:
```
RUN mkdir -p build
WORKDIR /src/build/
RUN CXX=g++ CC=gcc cmake ..
```

But I suggest you put the weight file somewhere else rather than just drop it in root dir, e.g. src/tests/weights/

Reply:
What does this code do?

```
RUN mkdir -p /src/
WORKDIR /src/
COPY . /src/
```

It looks like the copy will be from/to the same place?